src/open_inwoner/search/tests/test_page.py

```diff
@@ -294,26 +302,60 @@ def test_search_with_filters(self):
         # check if we see all checkboxes
         for facet in FacetChoices.values:
             # tags, organizations, categories
-            controls = page.locator(f"input[form='search-form'][name='{facet}']")
-            expect(controls).to_have_count(2)
-            for checkbox in controls.all():
+            mobile_controls = page.locator(".filters--desktop").locator(
+                f"input[form='search-form'][name='{facet}']"
+            )
+            expect(mobile_controls).to_have_count(2)  # only controls on desktop
+
+            # Check desktop controls.
+            for checkbox in mobile_controls.all():
                 # the input elements are hidden for styling so just test for enabled
                 expect(checkbox).to_be_enabled()
                 expect(checkbox).not_to_be_checked()
 
-        def _click_checkbox_for_name(page, name):
+            # Check mobile controls
+            mobile_controls = page.locator(".filters--mobile").locator(
+                f"input[form='search-form'][name='{facet}']"
+            )
+            expect(mobile_controls).to_have_count(2)  # only controls on desktop
+
+            # Check desktop controls.
+            for checkbox in mobile_controls.all():
+                # the input elements are hidden for styling so just test for enabled
+                expect(checkbox).not_to_be_enabled()
+                expect(checkbox).not_to_be_checked()
+
+        def _open_filter_for_checkbox(filter):
+            filter_button = filter.locator(".filter__opener")
+
+            # Check if filter is not open yet.
+            if filter_button.get_attribute("aria-expanded") != "true":
+                filter_button.click()
+
+            # Filter should be open.
+            expect(filter_button).to_have_attribute("aria-expanded", "true")
+
+        def _click_checkbox_for_name(page=page, name=""):
+            filter = page.locator(".filters--desktop").locator(".filter", has_text=name)
+            # first we need to open the list so that all the checkboxes are visible.
```

Comment:
Meaning of the comment: do "we" need to open it? Or Playwright needs to check for opened list?